site/en/docs/privacy-sandbox/fledge-api/index.md

```diff
@@ -34,6 +34,16 @@ the browser, to choose relevant ads for websites the user has previously visited
 FLEDGE is the first experiment to be implemented in Chromium within the
 [TURTLEDOVE](https://github.com/WICG/turtledove) family of proposals.
 
+### Can the Topics API be used with the FLEDGE API? 
+
+Yes. An observed topic for the current user, provided by the Topics API, could be used as 
+contextual information by a seller or bidder. A topic could be included in
+the following properties:
+
+*  `auctionSignals`, a property of the auction configuration object passed to `navigator.runAdAuction()`
+*  `userBiddingSignals`, a property of the interest group configuration
+   object passed to `navigator.joinAdInterestGroup()`
+
```

Comment:
This is too high up in this document, which gives the information a little too much priority. Can you move this to a accordion section between "[What is the difference between FLEDGE and TURTLEDOVE?](https://pr-5381-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/fledge-api/#what-is-the-difference-between-fledge-and-turtledove)" and "[Available browser configuration](https://pr-5381-static-dot-dcc-staging.uc.r.appspot.com/docs/privacy-sandbox/fledge-api/#available-browser-configuration)"?

Also, add a link to the Topics docs to the first mention of Topics API.

Reply:
Done.